### PR TITLE
testing/tclx: new aport

### PR DESCRIPTION
**Pull request by snoe925**
Enhances the Alpine Tcl with Tcl extensions for making tcl scripting easier.

### CODE REVIEW

File: main/tclx/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	esac
+	cd "$_builddir"
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--localstatedir=/var \
+                --with-tcl=/usr/lib \
```
Comment (fabled) [inline main/tclx/APKBUILD]: Extra whitespace.

File: main/tclx/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+source="http://downloads.sourceforge.net/project/tclx/TclX/$pkgver/$pkgname$pkgver.tar.bz2
+	"
+
+_major=${pkgver%.*}
+_builddir="$srcdir"/tclx${_major}
+
+build() {
+	local _64bit="--disable-64bit"
+	case "$CARCH" in
+		x86_64) _64bit="--enable-64bit";;
```
Comment (fabled) [inline main/tclx/APKBUILD]: We normally prefer to indent case clauses at same level with the case. Though, this type of indentation has been on some other aports too.

File: main/tclx/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+license="BSD"
+depends="tcl"
+depends_dev="tcl-dev"
+makedepends="tcl-dev"
+subpackages="$pkgname-doc $pkgname-dev"
+source="http://downloads.sourceforge.net/project/tclx/TclX/$pkgver/$pkgname$pkgver.tar.bz2
+	"
+
+_major=${pkgver%.*}
+_builddir="$srcdir"/tclx${_major}
```
Comment (fabled) [inline main/tclx/APKBUILD]: Rename this to `builddir` this is now used instead. It also enables e.g. default_prepare to apply patches listed in $source.

**Comment by snoe925** [comment]
I moved this to community because it is similar to perl and python language libraries in community.

File: main/tclx/APKBUILD (hunk 3 above)
Comment (snoe925) [inline main/tclx/APKBUILD]: Changed

File: main/tclx/APKBUILD (hunk 2 above)
Comment (snoe925) [inline main/tclx/APKBUILD]: Changed

File: main/tclx/APKBUILD (hunk 1 above)
Comment (snoe925) [inline main/tclx/APKBUILD]: Changed

**Pull request closed without merging**